test/Escrow.js

```diff
@@ -143,63 +180,85 @@ contract('Escrow', (accounts) => {
 
     it('sets status to pending', async () => {
       await HMT.transfer(Escrow.address, 100, { from: accounts[0] });
-      await Escrow.setup(reputationOracle, recordingOracle, 1, 1, recOIpns, repOIpns, url, hash, { from: accounts[0] });
-      const status = await Escrow.getStatus.call();
+      await Escrow.setup(
+        reputationOracle,
+        recordingOracle,
+        1,
+        1,
+        recOIpns,
+        repOIpns,
+        url,
+        hash,
+        { from: accounts[0] }
+      );
+      const status = await Escrow.getStatus();
       assert.equal(1, status);
```

Comment:
Here as well to `actual`, `expected`

Reply:
Updated in [latest commit](https://github.com/hCaptcha/hmt-escrow/pull/130/commits/a276f2f8624e8aeb7be4266173fa33bb70c910dd#diff-6639d32f4f91562f4eec97bd924b494eR195) :handshake: